examples/distributed_data_classification_examples/test_models.ipynb

```diff
@@ -0,0 +1,373 @@
+    "    classifier = DomainClassifier(\n",
+    "        model_file_name=model_file_name,\n",
+    "        labels=domain_labels,\n",
+    "        batch_size=1024,\n",
+    "    )\n",
+    "elif classifier_type == \"QualityClassifier\":\n",
+    "    quality_labels = [\"High\", \"Medium\", \"Low\"]\n",
+    "    model_file_name = \"/home/nfs/syurick/LLM_quality_classifier_inference/\" + \\\n",
+    "                      \"quality_rnd3_2014val1070_10ep_2xhigh_1024_fold4_last-001.pth\"\n",
+    "\n",
+    "\n",
+    "    classifier = QualityClassifier(\n",
+    "        model_file_name=model_file_name,\n",
+    "        labels=quality_labels,\n",
+    "        batch_size=1024,\n",
+    "    )\n",
+    "else:\n",
+    "    raise ValueError(\"Invalid classifier type\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run the actuall Classifier"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Starting domain classifier inference\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "GPU: 0, Part: 8: 100%|██████████| 937/937 [00:15<00:00, 62.45it/s]]]\n",
+      "GPU: 0, Part: 9: 100%|██████████| 937/937 [00:13<00:00, 68.83it/s] \n",
+      "GPU: 0, Part: 5: 100%|██████████| 938/938 [00:13<00:00, 67.67it/s] \n",
+      "GPU: 0, Part: 15: 100%|██████████| 937/937 [00:15<00:00, 62.42it/s] \n",
+      "GPU: 0, Part: 10: 100%|██████████| 937/937 [00:14<00:00, 64.39it/s] \n",
+      "GPU: 0, Part: 3: 100%|██████████| 938/938 [00:13<00:00, 67.37it/s] \n",
+      "GPU: 0, Part: 7: 100%|██████████| 937/937 [00:14<00:00, 66.83it/s] \n",
+      "GPU: 0, Part: 11: 100%|██████████| 937/937 [00:14<00:00, 65.50it/s]\n",
+      "GPU: 0, Part: 2: 100%|██████████| 938/938 [00:13<00:00, 68.84it/s]\n",
+      "GPU: 0, Part: 6: 100%|██████████| 938/938 [00:14<00:00, 64.48it/s]\n",
+      "GPU: 0, Part: 4: 100%|██████████| 938/938 [00:14<00:00, 63.31it/s]\n",
+      "GPU: 0, Part: 13: 100%|██████████| 937/937 [00:14<00:00, 63.18it/s]\n",
+      "GPU: 0, Part: 12: 100%|██████████| 937/937 [00:14<00:00, 62.92it/s]\n",
+      "GPU: 0, Part: 0: 100%|██████████| 938/938 [00:15<00:00, 62.14it/s]\n",
+      "GPU: 0, Part: 14: 100%|██████████| 937/937 [00:15<00:00, 62.04it/s]\n",
+      "GPU: 0, Part: 1: 100%|██████████| 938/938 [00:16<00:00, 56.99it/s]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Writing to disk complete for 16 partitions\n",
+      "CPU times: user 4.61 s, sys: 6.09 s, total: 10.7 s\n",
+      "Wall time: 39.8 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "\n",
+    "result_dataset = classifier(dataset=input_dataset)\n",
+    "result_dataset.df = result_dataset.df.rename(columns={\"labels\": f\"{classifier_type}_prediction\"})\n",
```

Comment:
Doesn't the `"labels"` column get renamed automatically to `pred_column` set in the constructor of the classifier?

Reply:
Removed, was left by mistake